axelor-account/src/main/java/com/axelor/apps/account/web/FixedAssetController.java

```diff
@@ -273,4 +276,40 @@ public void failOverControl(ActionRequest request, ActionResponse response)
     FixedAsset fixedAsset = request.getContext().asType(FixedAsset.class);
     Beans.get(FixedAssetFailOverControlService.class).controlFailOver(fixedAsset);
   }
+
+  public void setDepreciationPlanSelect(ActionRequest request, ActionResponse response) {
+    try {
+      FixedAsset fixedAsset = request.getContext().asType(FixedAsset.class);
+      if (fixedAsset.getFixedAssetCategory() != null
+          && fixedAsset.getFixedAssetCategory().getFixedAssetType() != null
+          && fixedAsset.getFixedAssetCategory().getFixedAssetType().getTechnicalTypeSelect()
+              == FixedAssetTypeRepository
+                  .FIXED_ASSET_CATEGORY_TECHNICAL_TYPE_SELECT_ONGOING_ASSET) {
+        fixedAsset.setDepreciationPlanSelect(
+            FixedAssetCategoryRepository.DEPRECIATION_PLAN_SELECT_NONE);
+      }
+      if (StringUtils.notEmpty(fixedAsset.getDepreciationPlanSelect())) {
+        response.setValue("depreciationPlanSelect", fixedAsset.getDepreciationPlanSelect());
+      }
```

Comment:
Same issue as FixedAssetCategoryController.java both methods should call the same service.